QUALITY_DECLARATION.md

```diff
@@ -29,11 +29,11 @@ The change history can be found in the [release notes section](https://iceoryx.i
 
 All symbols in the installed headers are considered part of the public API.
 
-Except for the exclusions listed below, all installed headers are in the include directory of the package, headers in any other folders are not installed and considered private. Headers under the folder `internal` are not considered part of the public API and are subject to change without notice.
+Except for the exclusions listed below, all installed headers are in the include directory of the package, headers in any other folders are not installed and considered private. Headers in the folder `internal` are not considered part of the public API and are subject to change without notice.
 
 ### API Stability Within a Released ROS Distribution [1.iv]/[1.vi]
 
-`iceoryx` guarantees public API stability for MINOR and PATCH releases and can not guarantee public API stability for MAJOR releases.
+`iceoryx` guarantees public API stability for MINOR and PATCH releases and cannot guarantee public API stability for MAJOR releases.
```

Comment:
I think we usually just write iceoryx and not `iceoryx`. Can you please correct this here and in the whole file?